api/log.go

```diff
@@ -0,0 +1,66 @@
+/*
+   Copyright (c) 2016 VMware, Inc. All Rights Reserved.
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package api
+
+import (
+	"net/http"
+
+	"github.com/vmware/harbor/dao"
+	"github.com/vmware/harbor/models"
+	"github.com/vmware/harbor/utils/log"
+)
+
+//LogAPI handles request api/logs
+type LogAPI struct {
+	BaseAPI
+	userID int
+}
+
+//Prepare validates the URL and the user
+func (l *LogAPI) Prepare() {
+	l.userID = l.ValidateUser()
+}
+
+//Get returns the recent logs according to parameters
+func (l *LogAPI) Get() {
```

Comment:
@ywk253100  Yes. For example, you can send request like: 
/api/logs?start_time=2016-05-12 00:00:00&end_time=2016-05-13 00:00:00